desafio-02/flafmg/csharp/README.md

```diff
@@ -0,0 +1,24 @@
+# Desafio 02 - Primos
+
+Este programa lista todos os números primos entre 1 e 10.000 utilizando C#. feito por @flafmg
+
+## pré-requisitos
+
+- [.NET SDK 8.0](https://dotnet.microsoft.com/download/dotnet/8.0)
+
+## compilar e executar
+
+1. **Clone o repositório:**
+   ```bash
+   git clone https://github.com/OsProgramadores/op-desafios.git
```

Comment:
Estas não são as instruções para clonagem da **sua solução,** (seu fork) mas sim do **repositório original**, o qual você fez o fork. 

Isso é intencional?

Reply:
sim, caso eu acabe privando ou deletando meu fork

Reply:
botei pro meu fork

Reply:
Agora sim. Esse tipo de mudança na documentação (apontar para o repo original) só é feita na hora que você for privar/deletar o seu fork. Antes disso é totalmente desnecessário, fica fora do escopo.